storage/plugins/dewey-editor/DeweyEditorPlugin.php

```
@@ -0,0 +1,635 @@
+
+        $validator = new DeweyValidator();
+        $errors = $validator->validate($data);
+
+        if (empty($errors)) {
+            return $this->jsonSuccess($response, ['valid' => true, 'message' => __('Dati validi.')]);
+        }
+
+        return $this->jsonSuccess($response, ['valid' => false, 'errors' => $errors]);
+    }
+
+    public function exportData(Request $request, Response $response, array $args): Response
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)